src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -10,11 +10,19 @@
  */
 public class StringUtil {
     public static boolean containsIgnoreCase(String source, String query) {
-        String[] split = source.toLowerCase().split("\\s+");
-        List<String> strings = Arrays.asList(split);
+        List<String> strings = listifyString(source);
```

Comment:
These changes are in the other PR too. Please don't duplicate code like this across PRs